wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/addtowallet/AddToWalletScreen.kt

```diff
@@ -218,4 +285,46 @@ private fun AddToWalletScreenWithIssuerDisplayDatas(
             }
         }
     }
+
+    // Scan Credential Offer
+    FilledTonalButton(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(8.dp),
+
+        colors = ButtonColors(
+            containerColor = MaterialTheme.colorScheme.primaryContainer,
+            contentColor = MaterialTheme.colorScheme.onPrimaryContainer,
+            disabledContainerColor = Color.Green,
+            disabledContentColor = Color.Green
+        ),
+
+        shape = RoundedCornerShape(8.dp),
+        onClick = {
+            onShowScanQrDialog.invoke()
+        }) {
+        Row {
+            Image(
+                painter = painterResource(id = R.drawable.qr_icon),
+                contentDescription = stringResource(R.string.accessibility_artwork_for)
+                        + stringResource(id = R.string.credential_offer_details),
+                modifier = Modifier
+                    .size(48.dp)
+                    .padding(4.dp)
+            )
+            Column(
+                modifier = Modifier.align(Alignment.CenterVertically)
+            ) {
+                Text(
+                    text = stringResource(id = R.string.credential_offer_scan),
+                    style = MaterialTheme.typography.labelLarge
+                )
+                Text(
+                    text = stringResource(id = R.string.credential_offer_details),
+                    style = MaterialTheme.typography.labelSmall,
+                    color = MaterialTheme.colorScheme.onSecondary
```

Comment:
secondary, instead of onSecondary?
In the screenshots, this doesn't match any of the other buttons, and it's hard to read.